estate/models/estate_property.py

```diff
@@ -0,0 +1,100 @@
+from odoo import models, fields, api, _
+from dateutil.relativedelta import relativedelta
+from datetime import date
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_is_zero, float_compare
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Test Model"
+    _order = "id desc"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=lambda self: date.today() + relativedelta(months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facade = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West")
+        ],
+    )
+    state = fields.Selection(
+        selection=[
+            ("new", "New"),
+            ("offer_received", "Offer Received"),
+            ("offer_accepted", "Offer Accepted"),
+            ("sold", "Sold"),
+            ("canceled", "Canceled")
+            ],
+        required=True,
+        copy=False,
+        default='new'
+    )
+
+    # relation between models
+    property_type_id = fields.Many2one("estate.property.type")
+    buyer_id = fields.Many2one("res.partner", copy=False)
+    salesperson_id = fields.Many2one("res.users", string="Sales man", default=lambda self: self.env.user)
+    tags_ids = fields.Many2many("estate.property.tag")
+    offer_ids = fields.One2many("estate.property.offer", "property_id")
```

Comment:
I've made a mistake here, odoo will remove the _id or _ids when no string is given

So here the string will be "tags" but the fieldname should be tag_ids with no "plural" word

You do need to give these a string to state that they are plural

```suggestion
    tag_ids = fields.Many2many("estate.property.tag", string="Tags")
    offer_ids = fields.One2many("estate.property.offer", "property_id", string="Offers")
```